CONTRIBUTING.md

```diff
@@ -1,54 +1,50 @@
 # Want to contribute some code?
 
-## We’d really appreciate your support!
+We’d really appreciate your support! In order to keep things organized, please make sure you understand the following.
 
 - If the [Github issue](https://github.com/jamulussoftware/jamulus/issues) for your feature/bug fix already exists, write a message in that issue saying you want to work on it.
 - Otherwise, please [post on the GitHub Discussions](https://github.com/jamulussoftware/jamulus/discussions) and say that you are planning to do some coding and explain why. Then we can discuss the specification.
-- Please only start coding after we have agreed to a specification to avoid putting lots of work into something which may not get accepted later.
+- To avoid putting lots of work into something which may not get accepted later, please start coding after we have agreed to a specification.
```

Comment:
```suggestion
- To avoid putting lots of work into something which may not get accepted later, please only start coding after we have agreed to a specification.
```